bundle/Resources/config/services.yml

```diff
@@ -222,6 +222,8 @@ services:
     ezpublish_legacy.persistence_cache_purger:
         class: eZ\Bundle\EzPublishLegacyBundle\Cache\PersistenceCachePurger
         public: false
+        calls:
+            - [setContainer, ["@service_container"]]
         arguments:
             - "@ezpublish.cache_pool"
             - "@ezpublish.spi.persistence.cache.locationHandler"
```

Comment:
```suggestion
            - "@ezpublish.spi.persistence.cache.locationHandler"
            - "%ezpublish_legacy.clear_all_spi_cache_on_symfony_clear_cache%"
            - "%ezpublish_legacy.clear_all_spi_cache_from_legacy%"
```

Reply:
Fixed in d6e4d9d36fe2ee9291967ef14598f3aa3e83690e